addons/event/models/event_registration.py

```diff
@@ -270,6 +271,20 @@ def _update_mail_schedulers(self):
         # cron triggers are correctly used
         onsubscribe_schedulers.execute()
 
+    def action_unarchive(self):
```

Comment:
I think it would be safer to override the "toggle_active" method.
Indeed, the code might call it directly instead of going through action_unarchive.

As a general rule, when overriding ORM methods, you should check if "other people do it".
So here, I greped "action_unarchive", and actually there are no overrides.
But if you grep "toggle_active", you will see plenty of overrides.